src/components/carousel/carousel.spec.tsx

```diff
@@ -26,6 +29,25 @@ function renderCarousel(props: CarouselProps, renderer = mount) {
 }
 
 describe("Carousel", () => {
+  describe("Deprecation warning", () => {
+    it("when user uses the component, a deprecation warning is raised only once in the console", () => {
+      const loggerSpy = jest.spyOn(Logger, "deprecate");
+      mount(
+        <>
+          <Carousel />
+          <Carousel />
```

Comment:
Question: could you please tell me the reason of using <Carousel /> twice ?

Reply:
Its to check that even if there are multiple instances of the component, the deprecation warning on the console is only fired once - this is mainly to reduce spam for consumers.